### ISSUE TITLE
Should there be a status between "Active" and "Inactive"? Or an alias for "Inactive" like "Passive"?

### ISSUE THREAD

**Issue by trallnag** [jantman/repostatus.org#42]
Let's say you have a project that has reached a very stable status. No additional features are planned and activities are limited to bug fixes and chore updates. According to the definition by <repostatus.org> this would be an inactive project. To me this sounds kind of wrong. Something like "Passive" would fit better. Though some definitions say that "Passive" and "Inactive" are acronyms.

What do you think?

**Comment by waldyrious** [comment]
Agreed — the badge could be interpreted to mean either "This project happens to be inactive (but can come to action in the right circumstances)" or "This project has been deliberately placed in an inactive state". The latter meaning suggests much less openness to, e.g., accepting pull requests or fixing reported bugs, and indeed it sounds more like the "Abandoned" status.

Considering the "support/maintenance will be provided as time allows" part of the description, I'd be in favor of replacing "Inactive" with "Passive" or another similar term that better represents that status. Perhaps "Reactive" as a play on words with the existing "Active" status, could also work? Or "Stabilized"... dunno, just spitballing ideas :)

**Comment by Alarm-Siren** [comment]
Of the four options of "Inactive", "Passive", "Reactive" and "Stabilized", I think Passive best communicates the intent of the status.

I don't think the meaning of Reactive would be obvious to a passer-by, though I appreciate the play on words.

If I didn't know the statuses already, then Inactive to me would be imply Unsupported.

I'd like to suggest a pair of new options: "Maintenance" or "Supported". This is because whilst I like Passive, it kind-of implies some level of... sluggishness, like "yeah we'll get to your issue eventually if we feel like it". These two options more directly imply that the repo is still being looked after.

Another thought, a new status of "Deprecated", meaning the same as Inactive/Passive/Supported except with the additional valence of "but you should probably try to find something newer".
